src/drivers/uart_ttl_generic.cpp

```diff
@@ -8,24 +8,32 @@
 /*!
     TODO: Description of initialization function for uart_ttl_generic
 */
-void init_uart_ttl_generic(uint32_t profile_id, R_UART_TTL_Generic profile)
+bool init_uart_ttl_generic(uint32_t profile_id, R_UART_TTL_Generic profile)
 {
     if(profile.port == UART_Port_UART2){
         Serial2.begin(profile.baudrate);
+        return true;
```

Comment:
Oh, yes. I'll implement the `while` loop.
But I'm not sure about the fist if-clause: Since we cannot delete/deinitialize any interfaces on the MCU, we simply have to overwrite the old initialization.  I think we have to discuss this further on Wed.

Reply:
I see. 👍